src/cli/commands/init.js

```diff
@@ -141,6 +152,9 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
     }
 
     if (answer) {
+      if (manifestKey === 'private') {
```

Comment:
It would be a better design to add something like `dataType: 'boolean'` to the schema of questions when they are defined, then here check that data type and use "yn" in the case of booleans only.
Or perhaps even `inputFormatter: yn` in the questions schema, then here: `answer = question.inputFormatter ? question.inputFormatter(answer) : answer`